Intersect.Client/Maps/MapInstance.cs

```diff
@@ -651,6 +651,13 @@ public void DrawItemsAndLights()
             //Draw Map Items
             foreach (var item in MapItems)
             {
+                // Are we allowed to see and pick this item up?
+                if (!item.Value.VisibleToAll && !item.Value.Owner.Equals(Globals.Me.Id) && !Globals.Me.IsInMyParty(item.Value.Owner))
```

Comment:
Use `==` instead of `.Equals` unless you can't, or it's a string (strings should be compared with `string.Equals(a, b, [comparison type])` except in special compiled query cases).

Since Id/Owner are Guids and are non-nullable `.Equals` can't ever throw an NRE here, but there's no reason to not maintain consistency with the rest of the code base (the built-in `==` can't throw an NRE).